src/translations/en.js

```diff
@@ -76,6 +76,7 @@
 	"tooltip-compare-mode" : "Compare mode",
 	"tooltip-find-and-replace" : "Find and replace",
 	"tooltip-export-usfm" : "Export as USFM",
+	"tooltip-backup-usfm" : "Successfully Backed Up",
```

Comment:
Is this message still being used?

Reply:
This message has been changed on the branch issue45.